packages/components/src/spectrum/picker/Picker.tsx

```diff
@@ -14,17 +16,18 @@ import { usePickerProps } from './usePickerProps';
  * for the Spectrum Picker component.
  * See https://react-spectrum.adobe.com/react-spectrum/Picker.html
  */
-export function Picker({
-  UNSAFE_className,
-  ...props
-}: PickerProps): JSX.Element {
+export const Picker = React.forwardRef(function Picker(
+  { UNSAFE_className, ...props }: PickerProps,
+  ref: DOMRef<HTMLDivElement>
+): JSX.Element {
   const { defaultSelectedKey, disabledKeys, selectedKey, ...pickerProps } =
-    usePickerProps<PickerProps, HTMLDivElement>(props);
+    usePickerProps(props);
 
   return (
     <SpectrumPicker
       // eslint-disable-next-line react/jsx-props-no-spreading
       {...pickerProps}
+      ref={ref}
```

Comment:
I just realized the `ref` prop returned by `usePickerProps` needs to be incorporated here as it is used to wire up the scroll handling.

Reply:
Possible `useMappedRef` could be incorporated here to map to the forwardRef and the scrollRef

Reply:
This is going to be a little annoying, but looks like the ref originates in `usePopoverOnScrollRef` and will need to have it's type signature changed from `useRef<T>(null)` to `useRef<T | null>(null);` to make it mutable.